### PR TITLE
[Testcase Refactoring] Add proper hw_classification to test_building

### PR DESCRIPTION
**Pull request by TTTIAN998**
## Summary
- Add hw_classification annotations to test classes and aligning test methods with their hardware classification.

## Changes
- Add "GENERIC" hw_classification to TestOpRecorder in test/onnx/exporter/test_building.py.
- Import HardwareClassification from torch.testing._internal.common_utils.

## Test Plan
- python test/onnx/exporter/test_building.py.

## Notes
- This is part of the test case refactoring initiative tracked in [Test] PyTorch Test Case Refactoring and Track #185590.

### CODE REVIEW

File: test/onnx/exporter/test_building.py [hunk 1]
```diff
@@ -10,9 +10,11 @@
 import torch
 from torch.onnx._internal.exporter import _building, _tensors
 from torch.testing._internal import common_utils
+from torch.testing._internal.common_utils import HardwareClassification
 
 
 class TestOpRecorder(common_utils.TestCase):
+    hw_classification = HardwareClassification.GENERIC
     def setUp(self):
```
Comment (lizhihai137) [inline test/onnx/exporter/test_building.py:19]: Add a blank line between the label attribute and the function.

Comment (TTTIAN998) [reply test/onnx/exporter/test_building.py:19]: Fixed.

**Review by lizhihai137 — APPROVED** [review]

**Review by orangeH25 — APPROVED** [review]

**Review by fffrog — APPROVED** [review]
SGTM
